README.md

```diff
@@ -1,72 +1,67 @@
 [![Build Status](https://travis-ci.org/jmaicaaan/express-starter-ts.svg?branch=master)](https://travis-ci.org/jmaicaaan/express-starter-ts)
 
 # express-starter-ts
-This repository gives the developer an Express Starter Typescript to kick off there development fast
+This repository gives the developer an Express Starter written in Typescript to kick off there development fast 🔥🔥🔥
 
+This is inspired from the [Java - Struts Framework](https://javabeat.net/introduction-to-struts-actions/) which we customized to make an opinionated starter pack for the developer to kick off there development fast.
 
-# Overview
-  - This is inspired from the [Java - Struts Framework](https://javabeat.net/introduction-to-struts-actions/) which we customized to make an opinionated starter pack for the developer to kick off there development fast.
-  - We also promote the `Single Responsibility` principle as we believe that it will be easier to maintain and provide unit tests when it scales up.
+We also promote the `Single Responsibility` principle as we believe that it will be easier to maintain and provide unit tests when it scales up.
 
-    ![flow](https://jmaicaaan.github.io/express-starter-ts/images/flow.jpg)
+  ![flow](https://jmaicaaan.github.io/express-starter-ts/images/flow.jpg)
```

Comment:
can we add this issue on next branch? then replace this the updated flow. But if we will put this on current master, this is ok now

Reply:
Nope, for now this is not included in the next branch. The next branch is only created this week and may cause some bugs still. We can alter this as soon we make the next branch official